client/src/components/pages/MainChannel.tsx

```diff
@@ -102,11 +103,18 @@ const ChannelsFragment: FC<ChannelProps> = ({channel, searchArgs}) => {
   const [bannerError, setBannerError] = useState<boolean>(false);
   const orientation = useOrientation();
   const navigation = useNavigation<MainStackNavigationProps<'MainTab'>>();
+  const {showModal} = useLeaveChannelContext();
 
   const onEndReached = (): void => {
     loadNext(ITEM_CNT);
   };
 
+  const showLeaveModal = (channelId: any): void => {
```

Comment:
#### Must Fix
We should avoid using `any`. Please use an actual type here.

Reply:
I'll modify it to the string type.